server/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "server",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@sonatype/react-shared-components": "file:../lib/dist",
```

Comment:
Note: This cannot use `link:` because node follows symlinks when importing.  What this means is that when an RSC module imports, say, `react`, even though from next's perspective that RSC module is at `server/node_modules/@sonatype/react-shared-components/...`, and you'd want it to resolve to `server/node_modules/react`, the `react-shared-components` folder there is a symlink to `lib/dist` and so node will actually resolve the react import to `lib/node_modules/react`.  This leads to "duplicate react instances" errors when using any hooks.

Of course, in a real app that isn't hosted in the same repo as the RSC source code, you wouldn't be using either `link:` or `file:`, so this doesn't really matter outside of our own testing